services/src/pro/mod.rs

```diff
@@ -13,3 +13,8 @@ pub mod tasks;
 pub mod users;
 pub mod util;
 pub mod workflows;
+
+// re-export test macro
+pub mod getest {
```

Comment:
maybe spend a few letters more in the name to make clear what it is?

Reply:
[749ac1c](https://github.com/geo-engine/geoengine/pull/859/commits/749ac1c0fbf842a215ed0a07396c3204a9c018b6)